dspace-api/src/test/java/org/dspace/content/RelationshipMetadataServiceTest.java

```diff
@@ -138,5 +138,41 @@ public void testGetRelationshipMetadata() throws Exception {
         assertThat(list.get(1).getMetadataField().getElement(), equalTo("isAuthorOfPublication"));
         assertThat(list.get(1).getAuthority(), equalTo("virtual::" + relationship.getID()));
 
+    }
+
+    @Test
+    public void testGetNextRightPlace() throws Exception {
```

Comment:
Could we add a corresponding test for `findNextLeftPlaceByLeftItem`?  I just noticed it seems to be missing.